pkgs/tools/admin/wander/default.nix

```diff
@@ -1,17 +1,33 @@
-{ buildGoModule, fetchFromGitHub, lib }:
+{ buildGoModule, fetchFromGitHub, installShellFiles, lib }:
 
 buildGoModule rec {
   pname = "wander";
-  version = "0.7.0";
+  version = "0.8.0";
+
+  # wander embeds commitSHA into version information
+  commitSHA = "94e61b57f353b7b28abf9c9d088319a2d98ecbfa";
```

Comment:
```suggestion
```
The bot won't be able to update this commitSHA unless if you make an update script. But then, you'd increase the upkeep burden for maintainers (update scripts are messy). I suggest you leave it out.

Reply:
I have removed it, but had to patch `wander` (unset commitSHA would panic at startup)